core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -141,7 +140,9 @@ private static void onResponse(
 
         final int childrenSize = log.children().size();
         if (childrenSize > 0) {
-            metrics.actualRequests().increment(childrenSize);
+            updateRetryingClientMetrics(ctx, log,
+                                        new DefaultClientRequestMetrics(registry, idPrefix, true),
```

Comment:
I don't think we want to instantiate a new `DefaultClientRequestMetrics` just for retrying.
I left a comment at https://github.com/line/armeria/pull/4707#discussion_r1145667481, so let's see how the other maintainers think 😄